### ISSUE TITLE
NTR: 5777-8 Cholesterol crystals

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4853]
**Preferred term label:**
Cholesterol crystals

**Definition (free text, please give PubMed ID)**
Colorless rectangular plates with notched corners (always observed with fat), associated with chyluria, UTI, or nephrotic syndrome.
https://details.loinc.org/LOINC/5777-8.html

**Parent term (use hpo.jax.org/app)**
HP_0031203 'Fatty casts'
(not sure if this is the best parent)

**Comment by pnrobinson** [comment]
This LOINC is for a urinary phenotype.
see also https://www.ncbi.nlm.nih.gov/pubmed/8709747

**Comment by nicolevasilevsky** [comment]
Should I annotate this to HP_0000079 'Abnormality of the urinary system'?

**Comment by pnrobinson** [comment]
Don't understand -- annotate what? Probably we need two separate terms.

**Comment by nicolevasilevsky** [comment]
This is for LOINC2HPO - there is a LOINC code for Cholesterol crystals, so should we create a new HPO term for Cholesterol crystals and then I can annotate the LOINIC code with that term?

**Comment by pnrobinson** [comment]
Cholesterol crystals can occur at multiple places in the body and so "Cholesterol crystals" would be ambiguous. I will try to research this!

**Comment by nicolevasilevsky** [comment]
Thank you, @pnrobinson!

**Comment by pnrobinson** [comment]
adding Cholesterol crystalluria as HP_0020176
